### ISSUE TITLE
Peer review by Angelica Ferlin (s306579)

### ISSUE THREAD

**Issue by angelicaferlin** [FlavioPatti/Computational-Intelligence_2022-23#4]
Overall a good solution, good job! I think you have found a good algorithm for the problem.

- Good results and a very fast solution. I am impressed that you managed to compute for N=100000.
- Typo: "#converte genome into a singol list".
- It is good that you have the powerpoint but your algorithm could be described in the same way in the readme as well.
- Choose one naming convention, you have one function called "gen2List(genome)" and another called "one_cut_cross_over". I would suggest changing the first one to "genome_to_list(genome)".
- In the uniform_cross_over() - what is the genoma? Is it the same as the genome?
- Overall - clean the code from unnecessary typos and comments like #print(), etc.
- I read Leonor Gomez peer review and would agree with her that it would be interesting to see if the result would change by using both crossovers according to a probability.

**Comment by FlavioPatti** [comment]
Yes ahah, "genoma" is the italianized version of the word "genome".
Thank you very much for the review, I really appreciated it!
